benchmarks/access_overhead/access_overhead_teams.cpp

```diff
@@ -130,7 +130,7 @@ struct Access<ViewType_t, typename std::enable_if_t<!std::is_same<
                                ? first_i + iters_per_team + iters_per_team_mod
                                : first_i + iters_per_team;
     Kokkos::parallel_for(Kokkos::TeamThreadRange(thread, first_i, last_i),
-                         [=](const StreamIndex i) { v(i) += 1; });
+                         [=, this](const StreamIndex i) { v(i) += 1; });
```

Comment:
That does not look right.

Reply:
What in particular?

Reply:
why did you add it?

Reply:
C++20 requires explicit capture of `this`. Note that this is a inner-nested `parallel_for` and capturing by value results in capturing a valid `this` ptr.